src/engine/animation.hpp

```diff
@@ -29,12 +29,13 @@ class Animation {
 
   u32 get_part_count();
 
-  /** Returns an AnimFrame from the current timestamp */
-  /** frames should be a pointer to the start of memory where there is exactly
-   * enough memory allocated for x AnimFrames, where x is the number of parts in
-   * the animation */
-  void get_anim_frames(u32 updates, AnimFrame frames[]);
+  /** Returns an AnimFrame from the current timestamp
+   *
+   * @param[out] frames This should be a pointer to the start of enough allocated memory for all AnimFrames
```

Comment:
over 80 chars + make sure it's documented that the amount of anim frames == number of parts